src/main.js

```diff
@@ -4,25 +4,25 @@ var pastActivities = [];
 var studyButton = document.querySelector('.study');
 var meditateButton = document.querySelector('.meditate');
 var exerciseButton = document.querySelector('.exercise');
+var catButtons = document.querySelectorAll('.cat-button')
```

Comment:
We should probably take one last look at our queries and make sure we are using them all and using them in more than one place. Especially since we did all that refactoring!